rack-box/scripts/install.sh

```diff
@@ -3,7 +3,7 @@
 # Exit if anything goes wrong
 
 set -eo pipefail
-export USER=ubuntu
+export USER=${USER:-ubuntu}
```

Comment:
I think it would be clearer (and safer) to pass USER on the script's command line as an optional argument.  Please replace this line with `export USER=${1:-ubuntu}` which means if an argument is passed, use that argument instead of `ubuntu` as USER's value.

Reply:
Agreed, will do.